DXMainClient/Domain/Multiplayer/MapLoader.cs

```diff
@@ -55,6 +88,137 @@ public void LoadMapsAsync()
             thread.Start();
         }
 
+        /// <summary>
+        /// Start the file watcher for the custom map directory.
+        ///
+        /// This will refresh the game modes and map lists when a change is detected. 
+        /// </summary>
+        public void StartCustomMapFileWatcher()
+        {
+            customMapFileWatcher = new FileSystemWatcher(SafePath.CombineDirectoryPath(ProgramConstants.GamePath, CustomMapsDirectory));
+
+            customMapFileWatcher.Filter = $"*{MAP_FILE_EXTENSION}";
+            customMapFileWatcher.NotifyFilter = NotifyFilters.Attributes
+                                 | NotifyFilters.CreationTime
+                                 | NotifyFilters.DirectoryName
+                                 | NotifyFilters.FileName
+                                 | NotifyFilters.LastAccess
```

Comment:
Why monitoring `LastAccess`?

Reply:
Also why don't just use the default NotifyFilter

Reply:
I was originally thinking we'd need to reload the map every time Final Alert saved it, but I realized that doesn't matter because the game just loads the current version of the file. Guess I forgot to pull this.